.github/workflows/salus.yaml

```
@@ -0,0 +1,19 @@
+on: [push]
+
+jobs:
+  salus_scan_job:
+    runs-on: ubuntu-latest
+    name: Salus Security Scan
+    steps:
+    - uses: actions/checkout@v1
```

```suggestion
    - uses: actions/checkout@v2
```
`v2` is the latest https://github.com/actions/checkout

---

Thanks for the suggestion!